src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -3297,7 +3297,7 @@ export class StudyViewPageStore {
                     record.qValue === undefined ? '' : getQValue(record.qValue),
                     record.totalCount,
                     record.numberOfAlteredCases,
-                    getFrequencyStr(record.numberOfAlteredCases / record.numberOfSamplesProfiled * 100)
```

Comment:
@alisman I an worried I did the wrong thing here. When I ran `generateAPI`, the changes broke everything you see me fix in the commit. I looked at the type decl and it looked like it was just a rename, but I have no real clue.